backends/graphics/surfacesdl/surfacesdl-graphics.cpp

```diff
@@ -1130,8 +1150,9 @@ void SurfaceSdlGraphicsManager::internUpdateScreen() {
 		}
 
 		SDL_LockSurface(srcSurf);
+#ifndef PSP2
```

Comment:
This is kind of ugly, so I'd like to clarify, what are the consequences of locking the _hwscreen on PSP2? Since you seem to be OK with SDL_LockSurface(srcSurf) above.